components/elm/src/biogeochem/CNNitrogenFluxType.F90

```diff
@@ -1078,7 +1078,7 @@ subroutine Summary(this, bounds, num_soilc, filter_soilc, num_soilp, filter_soil
     ! !USES:
     use elm_varpar    , only: nlevdecomp,ndecomp_cascade_transitions,ndecomp_pools
     use subgridAveMod , only: p2c
-    use pftvarcon     , only : npcropmin 
+    use pftvarcon     , only : iscft
```

Comment:
this can be removed since its not used, right?

Reply:
Yes - removed in latest commit.